src/desktop/apps/auction/routes.js

```diff
@@ -36,9 +36,9 @@ export async function index (req, res, next) {
     let articles = []
 
     try {
-      ({ articles } = await metaphysics({
+      ;({ articles } = await metaphysics({
```

Comment:
@sweir27 - yeah, leading parens can be weird because when everything is compiled / minified (when not using semis) something like 

```javascript
const foo = []

({ articles }) = ... would look like an empty function call:

const foo = []({ articles })
```

Mostly no semis is fine but there are a few little cases that one has to be cautious of